app/MomMock/Entity/Rma/Item.php

```diff
@@ -0,0 +1,86 @@
+<?php
+/**
+ * Copyright (c) 2018 Magenerds
+ * All rights reserved
+ *
+ * This product includes proprietary software developed at Magenerds, Germany
+ * For more information see http://www.magenerds.com/
+ *
+ * To obtain a valid license for using this software please contact us at
+ * info@magenerds.com
+ */
+
+namespace MomMock\Entity\Rma;
+
+use MomMock\Entity\AbstractEntity;
+
+/**
+ * Class Item
+ * @package MomMock\Entity\Order
+ * @author  Florian Sydekum <f.sydekum@techdivision.com>
+ */
+class Item extends AbstractEntity
+{
+    /**
+     * Item status
+     */
+    const STATUS_PENDING = 'requested';
+    const STATUS_COMPLETE = 'approved';
+    const STATUS_CANCELLED = 'cancelled';
+
+    /**
+     * Holds the table name
+     */
+    const TABLE_NAME = 'rma_item';
+
+    /**
+     * Id field.
+     */
+    const ID_FIELD = 'return_id';
+
+    /**
+     * @var []
+     */
+    private $data;
+
+    /**
+     * @param [] $data
+     * @return Item
+     */
+    public function setData($orderId, array $data)
+    {
+        $this->data = $data;
+        $this->data[self::ID_FIELD] = $orderId;
+
+        return $this;
+    }
+
+    /**
+     * Saves an order item
+     *
+     * @return string
+     */
+    public function save()
+    {
+        $this->db->createQueryBuilder()
+            ->insert(sprintf("`%s`", self::TABLE_NAME))
+            ->values([
+                '`' . self::ID_FIELD . '`' => "'{$this->data[self::ID_FIELD]}'",
```

Comment:
You have an `ID_FIELD` which is `return_id`. In the SQL setup script you also have an `id` field. Which is correct?